.circleci/config.yml

```diff
@@ -36,21 +36,77 @@ jobs:
           command: |
             TEST_FILES="$(circleci tests glob "./test/**/*.ts" | circleci tests split)"
             yarn test ${TEST_FILES}
-  lint-and-test:
+  coverage-parallel:
     docker:
       - image: "cimg/base:stable"
-    resource_class: small
+    resource_class: large
+    parallelism: 8
     steps:
+      - checkout
+      - node/install:
+          install-yarn: true
+          lts: true
+      - node/install-packages:
+          pkg-manager: yarn
+      - run:
+          name: coverage
+          command: |
+            TEST_FILES="{$(circleci tests glob "./test/**/*.ts" | circleci tests split | xargs | sed -e 's/ /,/g')}"
+            yarn coverage --testfiles "${TEST_FILES}"
       - run:
-          name: Success
-          command: echo "Success"
+          name: save coverage
+          command: |
+            mkdir -p /tmp/coverage
```

Comment:
is there any cleanup we should do with this tmp dir?

Reply:
I'm not 100% sure, but I think circleci should clean up everything (including the persisted workspace) after the entire workflow is completed. https://circleci.com/docs/workspaces/

I'm ~copying the pattern in this repo: https://github.com/SetProtocol/set-v2-strategies/blob/29db4a3de4a6ecd55a2c86d85a002b7e1106f8b2/.circleci/config.yml#L79-L143

Reply:
Yeah that seems correct to me – also consistent w/ this (from those docs):

> The workspace is additive only, jobs may add files to the workspace but cannot delete files from the workspace.